dcat/index.html

```diff
@@ -3377,46 +3458,25 @@ <h2>Resource life-cycle</h2>
-
-<p>Compared with the second option, the first one may have the advantage of simplifying metadata management, and avoiding the creation of datasets having the same values for almost all their metadata elements. On the other hand, this approach reduces the ability of being discovered, as distribution metadata are not rich as datasets' ones. Moreover, using distributions may result cumbersome or unfeasible when the number of child datasets is too high.</p>
-
-<p>As stated in the note to <a href="#Class:Distribution"></a>, DCAT recommends, as the default approach, typing both dataset series and child datasets as <code>dct:Dataset</code>'s, and linking them by using properties <code>dct:hasPart</code> and/or <code>dct:isPartOf</code>, and possibly soft-typing the dataset series via property <code>dct:type</code>.</p>
-
-<aside class="ednote">
-<p>The creation of a specific class for dataset series is under discussion.</p>
-<div class="issue" data-number="1272"></div>
-</aside>
-
-<p>The approach based on the use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.</p>
-
-<p>Here and in the following sections, guidance will focus on the default approach.</p>
-
-<aside class="example" id="ex-dataset-series-containment" title=""> 
-<p>In the following example, yearly budget data are grouped into a series. Both the series and the child datasets are typed as <code>dcat:Dataset</code>'s, and the series is soft-typed with property <code>dct:type</code>. The series and the datasets are linked by using properties <code>dct:hasPart</code> and <code>dct:isPartOf</code>.</p>
+<p>DCAT makes dataset series first class citizens of data catalogs by minting a new class <a href="#Class:Dataset_Series"><code>dcat:DatasetSeries</code></a>, defined as subclass of <a href="#Class:Dataset"><code>dcat:Dataset</code></a>. The dataset series and the child datasets are linked by using the [[DCTERMS]] property <a href="#Property:dataset_series_has_part"><code>dct:hasPart</code></a> and/or its inverse <a href="#Property:dataset_series_is_part_of"><code>dct:isPartOf</code></a>. Note that a dataset series can also be the child of another dataset series.</p>
+  <div class="issue" data-number="1307"> </div>
+  
+<aside class="example" id="ex-dataset-series-containment" title="Yearly budget datasets grouped into a series"> 
+<p>In the following example, yearly budget data are grouped into a series. The series is typed as <code>dcat:DatasetSeries</code>, the child datasets are typed as <code>dcat:Dataset</code>. The series and the datasets are linked by using both <code>dct:hasPart</code> and <code>dct:isPartOf</code>.</p>
```

Comment:
Is terminology 'child dataset' standard? I think 'member' or 'component' dataset might be better?

Reply:
The terminology comes from ISO 19115, where the relationship between series and the datasets in a series is seen as a hierarchy - also because in ISO 19115 a series can be a child of another series.

Reply:
Does it use the specific term 'child'?

Reply:
I cannot confirm it is explicitly used in ISO 19115, but this is the term commonly used for the relationship in the geo community. Moreover, the link from a dataset to the series is specified by a metadata field called "parent identifier".